backend/apps/sandbox/serializers.py

```
@@ -177,3 +177,38 @@ class Meta:
+
+    class Meta:
+        model = PipelineExecution
+        fields = [
+            "id",
+            "project",
+            "trigger_command",
+            "status",
+            "jobs",
+            "created_at",
+            "completed_at",
+        ]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)